fhir-persistence-cassandra/src/main/java/com/ibm/fhir/persistence/cassandra/CassandraPropertyGroupAdapter.java

```diff
@@ -76,4 +77,30 @@ public String getLocalDatacenter() {
             throw new IllegalArgumentException("Property group not configured " + PROP_LOCAL_DATACENTER);
         }
     }
-}
+
+    /**
+     * Should the payload be compressed
+     * @return
+     */
+    public boolean isCompress() {
+        try {
+            return propertyGroup.getBooleanProperty(PROP_COMPRESS, true);
```

Comment:
one thing robin and I discussed is whether this should be in the config at all.
for cassandra, we probably want it to be true to avoid line length limitations.
for COS / Azure BLOB we probably want it to be false since the cloud storage services compress and encrypt the contents themselves

Reply:
Yes, will remove from config because we'll always compress for Cassandra, but never for COS/Azure Blob.